WLogger/WeekLogs/WeekLogsViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WeekLogsViewModel.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/27/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+public class WeekLogsViewModel {
+
+    public let weekTotalTitle: String
+    
+    private let logsAmount = MutableProperty<String>("")
+    public let fetchLogsAmount: Action<AnyObject, String, RepositoryError>
+    
+    private let loggerImage = MutableProperty<UIImage?>(nil)
+    public let fetchLoggerImage: Action<AnyObject, UIImage?, NSError>
+    
+    public let weekLogsTableViewModel: WeekLogsTableViewModel
+    
+    public init(imageFetcher: ImageFetcher, logRepository: LogRepository) {
```

Comment:
The image fetcher could be replaced by `NSURL -> SignalProducer<UIImage, NSError>`